config/src/test/scala/com/typesafe/config/impl/ConfParserTest.scala

```diff
@@ -780,6 +781,45 @@ class ConfParserTest extends TestUtils {
     }
 
     @Test
+    def includeRequiredMissing() {
```

Comment:
should we test a third level of nesting (that required() does not affect the requiredness of includes inside the required file - it should always affect only the immediate include)